ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/RedHat.yml

```diff
@@ -52,11 +52,11 @@
 ##########################
 # Additional build tools #
 ##########################
-- name: Install additional build tools if NOT RHEL 8
+- name: Install additional build tools if NOT RHEL 8 AND RHEL9
   package: "name={{ item }} state=latest"
   with_items: "{{ Additional_Build_Tools_NOT_RHEL8 }}"
```

Comment:
Should we rename this variable to `Additional_Build_Tools_NOT_RHEL8Plus`?

Reply:
Yes that makes sense.